src/components/PopupWithForm.js

```diff
@@ -35,4 +39,12 @@ export default class PopupWithForm extends Popup {
       this._handleFormSubmit(this._getInputValues());
     });
   }
+
+  showLoading() {
```

Comment:
Great that you've implemented the logic for rendering loading via the class methods